.github/workflows/django.yml

```diff
@@ -8,13 +8,27 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.13]
 
+    services:
+      postgres:
+        image: postgres:15
```

Comment:
I think the last version is 16? And smilar to docker compose with only Postgres password we are ok, and better alpine